### PR TITLE
chore: isolate readme content from template

### PR DESCRIPTION
**Pull request by huxuan**

### CODE REVIEW

File: copier.yaml [hunk 1]
```diff
@@ -170,3 +170,19 @@ default_py:
...
+    [% if project_name == "Serious Scaffold Python" -%]
+      [% include 'includes/sample/readme_content.jinja' %]
+    [%- else -%]
+    ## title 1
+
+      content 1
+
+    ## title 2
+
+      content 2
```
Comment (msclock) [inline copier.yaml:184]: Usually, developers maybe are not ready to fill the readme content, how about considering to leave it blank.🤔

Comment (msclock) [reply copier.yaml:184]: Or just add an annotation here or give a choice to leave a blank, for example:
```md
# Fill the readme content here
```

**Review by msclock — COMMENTED** [review]
This is what I'm thinking how to prevent some repository items for consistent verification. Provide a series of samples is a good solution.👍

File: copier.yaml (hunk 1 above)
Comment (huxuan) [reply copier.yaml:184]: That is totally OK, actually I was thinking how to demostrate a multiline string here.

Comment (msclock) [reply copier.yaml]: But it's ok and althogh the readme content placeholder seams to have too many text description.

Comment (msclock) [reply copier.yaml]: > That is totally OK, actually I was thinking how to demostrate a multiline string here.

Ah,I just consider sometimes we want to skip everything for quick start.

Comment (huxuan) [reply copier.yaml]: PTAL, let me know if there is any misunderstand for me.

**Review by msclock — APPROVED** [review]

**Pull request merged**